### ISSUE TITLE
NTR, pulmo: Bronchocentric Granulomatosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6386]
Preferred term label:
Bronchocentric Granulomatosis

Synonyms

Definition (free text, please give PubMed ID)
Granulomatous inflammation that surrounds the larger airways, replacing bronchial walls and mucosa. In bronchocentric granulomatosis, the lumen of the airway contains necrotic debris, and palisaded histiocytes surround the lumen.

Parent term (use hpo.jax.org/app)
Pulmonary granulomatosis HP:0030250

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Sarcoidosis ORPHA:797 , Granulomatosis with Polyangiitis ORPHA:900,Loeffler endocarditis ORPHA:75566, Allergic Bronchopulmonary Aspergillosis ORPHA:1164

Your nano-attribution (ORCID)
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
def: Granulomatous inflammation that surrounds the bronchi and bronchioles *** (Is this OK-- I want to be more specific) ***, replacing bronchial walls and mucosa. In bronchocentric granulomatosis, the lumen of the airway contains necrotic debris, and palisaded histiocytes surround the lumen.

Should we refer to Fig 2 in the comment
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4681958/

**Comment by pnrobinson** [comment]
added
